docs/concepts/policy/pod-security-policy.md

```diff
@@ -207,37 +207,58 @@ podsecuritypolicy "permissive" deleted
 
 ## Enabling Pod Security Policies
 
-In order to use Pod Security Policies in your cluster you must ensure the
-following
+In order to use Pod Security Policies in your cluster:
 
-1.  You have enabled the API type `extensions/v1beta1/podsecuritypolicy` (only for versions prior 1.6)
-1.  [You have enabled the admission control plug-in `PodSecurityPolicy`](/docs/admin/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in)
-1.  You have defined your policies
+1. Enable the API type `extensions/v1beta1/podsecuritypolicy` (only for versions prior 1.6)
+1. [Enable the admission control plug-in `PodSecurityPolicy`](/docs/admin/admission-controllers/#how-do-i-turn-on-an-admission-control-plug-in)
+1. Create PodSecurityPolicy objects
+1. Authorize users, groups, or service accounts to perform the `use` verb on desired policies
 
 ## Working With RBAC
 
 In Kubernetes 1.5 and newer, you can use PodSecurityPolicy to control access to
 privileged containers based on user role and groups. Access to different
 PodSecurityPolicy objects can be controlled via authorization.
 
-Note that [Controller Manager](/docs/admin/kube-controller-manager/) must be run
-against [the secured API port](/docs/admin/accessing-the-api/), and must not
-have superuser permissions. Otherwise requests would bypass authentication and
-authorization modules, all PodSecurityPolicy objects would be allowed,
-and user will be able to create privileged containers.
+Before beginning to grant access to specific policies, ensure users and service accounts have not been given global access
+by running the following commands as a privileged user.
+
+Check anonymous users (should answer `no`):
+```shell
+kubectl auth can-i use podsecuritypolicies --as=system:anonymous --as-group=system:unauthenticated
+```
+
+Check all authenticated users (should answer `no`):
+```shell
+kubectl auth can-i use podsecuritypolicies --as=any-random-user --as-group=system:authenticated
+```
+
+Check all service accounts (should answer `no`):
+```shell
+kubectl auth can-i use podsecuritypolicies --as=system:serviceaccount:any-random-ns:any-random-sa
+```
```

Comment:
Are "any-random-user", "any-random-ns", "any-random-sa" builtin strings for check? Or we can use `--as=system:serviceaccount:whatever-i-dont-care` as an example? Some clarifications would be helpful.

Reply:
> Are "any-random-user", "any-random-ns", "any-random-sa" builtin strings for check?

No, it's supposed that a user will substitute them by a real values.

Reply:
> No, it's supposed that a user will substitute them by a real values.

Not really, the commands are meant to be run as-is to test access by the named groups

Reply:
So @tengqm was right that "some clarifications would be helpful".

I don't understand, how the command `kubectl auth can-i use podsecuritypolicies --as=system:serviceaccount:any-random-ns:any-random-sa` will "check all service accounts" then.